frontend/src/components/data-table/column-header.tsx

```diff
@@ -162,6 +166,52 @@ export const DataTableColumnHeader = <TData, TValue>({
       </DropdownMenuSub>
     );
   };
+  // render pinning options
+  const renderPinning = () => {
+    // if the column cannot be pinned, return null
+    if (!column.getCanPin()) {
+      return null;
+    }
+    return (
+      <DropdownMenuSub>
+        <DropdownMenuSubTrigger>
+          <PinIcon className="mr-2 h-3.5 w-3.5 text-muted-foreground/70" />
```

Comment:
if not too much trouble, i think maybe we can extract this className (since its used in other icons). 

something like:
```
const iconClassName = "mr-2 h-3.5 w-3.5 text-muted-foreground/70";
```